content/installation/java/Java-basic-installation.md

```diff
@@ -6,16 +6,16 @@ tags: "installation Java agent overview installation"
 
 To install the Java agent, complete the following steps:
 
-* Log in to the Contrast interface.
+* Log in to the Contrast UI.
 * Click the button in the top navigation bar to **Add Agent**.
 * Select the **Java** agent in the dropdown menu, and click the button to **Download Agent**. You may need to specify proxy authentication information required by your network before downloading the agent.
 
 <a href="assets/images/Download-java-agent.png" rel="lightbox" title="Download the Java agent"><img class="thumbnail" src="assets/images/Download-java-agent.png"/></a>
 
+* After download, add a JVM parameter to install the Java agent. 
 
-> **Note:** You must not rename the *contrast.jar* file. If the agent file name is changed, the agent will [fail to connect](troubleshooting-javainstall.html#class) with the Contrast application. 
- 
+> **Example:** `java -javaagent:contrast.jar -jar <<app-name>>.jar`
```

Comment:
Should we use double angle brackets i.e. `<<foo>>` to mean "replace with user value"? I believe I've used single angle bracket notation for this in other docs because I didn't know we had a convention, but I'll make sure to correct those going forward if double brackets is our convention

Reply:
No set convention. I'll update to single.